test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+  def setup
+    @base_title = "Ruby on Rails Tutorial Sample App"
+  end
+
+  test "should get home" do
+    get static_pages_home_url
+    assert_response :success
+    assert_select "title", "Home | #{@base_title}"
```

Comment:
For `"title"` use single quotes.
For next string is good, because it's the interpolation - `"#{code}"`